ci-scripts/setup-cluster.sh

```diff
@@ -123,6 +127,21 @@ EOF
     wait_for_entity_by_selector 300 openshift-pipelines pod app=tekton-pipelines-webhook
     kubectl -n openshift-pipelines wait --for=condition=ready --timeout=300s pod -l app=tekton-pipelines-webhook
 
+    info "Enable performance options"
+    chains_perf_options=""
+    if [ -n "$chains_kube_api_qps" ] || [ -n "$chains_kube_api_burst" ] || [ -n "$chains_threads_per_controller" ]; then
```

Comment:
IMO this is extra `if` which is not needed as you are checking individual variables below in individual ifs anyway.